src/transports/tcp/btcp.c

```diff
@@ -113,6 +113,7 @@ int nn_btcp_create (struct nn_ep *ep)
 
     /*  Allocate the new endpoint object. */
     self = nn_alloc (sizeof (struct nn_btcp), "btcp");
+    alloc_assert (self);
```

Comment:
You should remove the one below two lines down.

Reply:
Hi @gdamore! Thanks for the feedback. I removed the assert.